gitlab-pipeline/stage/build.yml

```diff
@@ -16,7 +16,7 @@ build:client:docker:
   needs:
     - init:workspace
   allow_failure: false
-  image: docker
+  image:  ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/docker
   services:
     - docker:dind
```

Comment:
Additionally, please add:
```
services:
    - name: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/docker:dind
      alias: docker

Reply:
Okey. We didn't add this on [the other PR](https://github.com/mendersoftware/mender/pull/1707/commits/4dcdc52510c943edc0db72ddc8d8c615380a563e), which I used as the reference.